docs/readme.md

```diff
@@ -0,0 +1,41 @@
+# WorkFlow Launcher Documentation
+
+**GitHub Pages [documentation website](https://broadinstitute.github.io/wfl)**
+
+## Rendering locally
+
+You may wish to launch a local version of our documentation website
+to test changes for formatting, link resolution, etc.
+
+Python 3 is requred:
+```
+cd /path/to/wfl/docs
+python3 pip install -r requirements.txt
+mkdocs serve
+```
+
+Logging output will indicate that changes are being detected,
+and point you to where documentation is served (e.g. http://127.0.0.1:8000).
+```
+INFO    -  Building documentation... 
+INFO    -  Cleaning site directory 
+WARNING -  A relative path to 'dev-sandbox.md' is included in the 'nav' configuration, which is not found in the documentation files 
+WARNING -  A relative path to 'dev-deployment.md' is included in the 'nav' configuration, which is not found in the documentation files 
```

Comment:
Didn't notice this until I ran it myself.

Reply:
I'm glad you mentioned this.  I thought that maybe I was missing some configuration on my end as a new person.  While I'm in here, I'll remove those undefined paths from the nav.